docs/vm/backup-restore.md

```diff
@@ -68,3 +68,42 @@ You can choose to either delete or retain the previous volumes. By default, all
 1. Click `Create`.
 
 The restore process can be viewed from the `Virtual Machines` page.
+
+## Restore a new VM on another Harvester cluster
+
+_Available as of v1.0.0_
+
+You can migrate a backup to another Harvester cluster and create a new VM based on it.
+
+**Requirements:** The VM image name and content must be the same as configured in the cluster where the backup was migrated from.
```

Comment:
prefer to use 
```
!!! prerequisites "Prerequisites"
     You must manually configure the virtual machine images with the same name on the new cluster first, otherwise the virtual machines will be failed to recover.
```

Reply:
Updated it. Thank you.